Runtime/SimpleGraphQL/HttpUtils.cs

```diff
@@ -21,7 +21,90 @@ public enum SubscriptionError
     [PublicAPI]
     public static class HttpUtils
     {
-        private static ClientWebSocket _webSocket;
+        public interface IWebSocket : IDisposable
+        {
+            public interface ISocketOptions
+            {
+                public void AddSubProtocol(string protocol);
```

Comment:
I don't think `public` is needed inside the interface 🙂

Reply:
Done